tests/test_plan.py

```
@@ -0,0 +1,294 @@
+import pytest
+import sys
+import asyncio
+from pathlib import Path
+
+project_root = Path(__file__).parent.parent
+if str(project_root) not in sys.path:
+    sys.path.insert(0, str(project_root))
+
+
+class TestExtractJson:
+    """测试 JSON 解析（复用 core.utils 的逻辑）"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)